### ISSUE TITLE
Crash at start

### ISSUE THREAD

**Issue by nake90** [mvysny/aedict#574]
Hello. Since the last 2 updates or so I get a crash at startup. My version is v3.38.3, android 4.1.2.
I got the following log:
```
10-07 09:50:48.523: V/ActivityThread(16973): Class path: /data/app/sk.baka.aedict3-2.apk, JNI path: /data/data/sk.baka.aedict3/lib
10-07 09:50:48.734: I/ActivityThread(16973): Pub sk.baka.aedict3.search: sk.baka.aedict3.search.SearchProvider
10-07 09:50:48.945: I/s*.b*.a*.AedictApp(16973): Storing the dictionaries to the following path: /storage/external_SD/Android/data/sk.baka.aedict3/files/aedict3/dictionaries
10-07 09:50:48.952: I/s*.b*.a*.c*.FileStorage(16973): SDCard Aedict root set to /storage/sdcard0/aedict3
10-07 09:50:48.960: I/s*.b*.a*.AedictApp(16973): AedictApp started in 211ms (prefs: 195ms, setRoot: 13ms, start-services: 3ms)
10-07 09:50:49.241: I/MainActivity(16973): Starting Aedict 3.38.3 en_US java.nio.charset.CharsetICU[UTF-8]
10-07 09:50:49.624: E/AndroidRuntime(16973): java.lang.RuntimeException: Unable to resume activity {sk.baka.aedict3/sk.baka.aedict3.search.MainActivity}: java.lang.IllegalArgumentException: value: blank
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict.util.Check.requireNotBlank(Check.java:64)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict.search.lucene.DBQuery$Term.<init>(DBQuery.java:287)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict.search.lucene.DBQuery.term(DBQuery.java:61)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict.search.lucene.DBQuery.jp(DBQuery.java:95)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict.search.JMDictQuery.globSearch(JMDictQuery.java:604)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict.search.JMDictQuery.forGenericEnJpTerm(JMDictQuery.java:505)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict.search.JMDictQuery.access$000(JMDictQuery.java:37)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict.search.JMDictQuery$Builder.forGenericEnJpTerm(JMDictQuery.java:414)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict3.util.Config.newQuery(Config.java:635)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict3.search.MainActivity.getSearch(MainActivity.java:400)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict3.search.MainActivity.liveSearch(MainActivity.java:419)
10-07 09:50:49.624: E/AndroidRuntime(16973): 	at sk.baka.aedict3.search.MainActivity.onResume(MainActivity.java:267)
```

It seems that there is an invalid search or something that gets executed every time I try to start the program. Is there a way to reset that so that it starts empty without having to lose all my information?

**Comment by mvysny** [comment]
Hi, I apologize for this crash. Thank you very much for the log, it helped tremendously. I will publish a new version shortly, which should hopefully remedy the crash.

**Comment by mvysny** [comment]
Should be fixed in Aedict 3.38.4

**Comment by nake90** [comment]
Thanks. The fix works perfectly.
Keep the good work :+1:
